ui-components/src/page_components/listing/listing_sidebar/LeasingAgent.tsx

```diff
@@ -53,25 +56,20 @@ const LeasingAgent = (props: LeasingAgentProps) => {
         </p>
       )}
 
+      {managementWebsite && (
+        <p className="my-5">
+          <a href={managementWebsite} target="_blank" rel="noreferrer noopener">
```

Comment:
What does the blank target do here?

Reply:
Apparently it makes it so that it opens in a new tab (I got that from the internet, e.g. https://stackoverflow.com/questions/15551779/open-link-in-new-tab-or-window); I added this on Tom's request in the original PR